requirements.txt

```diff
@@ -18,7 +18,7 @@ cssselect2==0.4.1
     # via svglib
 dj-database-url==0.5.0
     # via -r requirements.in
-django==3.1.13
+django==3.2.12
```

Comment:
3.2.13 is out so can you bump it again please?

https://docs.djangoproject.com/en/4.0/releases/3.2.13/

Reply:
Thanks Mark for the Hint! I will test everything and then i will push it.